blkmaker.c

```diff
@@ -664,9 +664,9 @@ bool _blkmk_insert_witness_commitment(blktemplate_t * const tmpl, unsigned char
 	
 	const size_t offset_of_txout_data = (offset_of_txout_count + in_txout_count_size);
 	const size_t new_offset_of_preexisting_txout_data = (offset_of_txout_count + out_txout_count_size);
-	const size_t length_of_txtail = 4;
+	const size_t length_of_txtail = 6; // lock time + EQB fields
```

Comment:
This code inserts the witness commitment before the lock time. Need to also move the EQB fields at the same time.